### PR TITLE
Improving the environment setup

### PR DESCRIPTION
**Pull request by alefcastelo**

### CODE REVIEW

File: docker-compose.cli.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: '3.7'
+
+services:
+    composer:
+        image: composer
+        working_dir: /application
```
Comment (ianrodrigues) [inline docker-compose.cli.yml]: Composer already have `/app` as working directory. Any special reason for using a new one?

Comment (alefcastelo) [reply docker-compose.cli.yml]: Nops, the working_dir for this image was '/', I'll change this after merge because I have other one PR based on this

Comment (alefcastelo) [reply docker-compose.cli.yml]: https://github.com/kdti/kdti-server/pull/117

Comment (ianrodrigues) [reply docker-compose.cli.yml]:
```bash
$ docker run --rm -it composer bash                                                                             
bash-4.4# pwd
/app
bash-4.4#
```

Comment (ianrodrigues) [reply docker-compose.cli.yml]: See: https://github.com/composer/docker/blob/master/1.9/Dockerfile#L60

**Comment by rtio** [comment]
It was merged in another PR

**Pull request closed without merging**